compiler/optimizer/Inliner.cpp

```diff
@@ -4821,8 +4817,11 @@ bool TR_InlinerBase::inlineCallTarget2(TR_CallStack * callStack, TR_CallTarget *
          dumpOptDetails(comp(), "Inliner: trees for %s\n", calleeSymbol->signature(trMemory()));
          comp()->dumpMethodTrees("after inlining while inlining", calleeSymbol);
          }
+
       }
 
+   getUtil()->requestAdditionalOptimizations(calltarget);
```

Comment:
Will down stream project see a regression from this until they update to this new mode of operating since the default implementation does nothing? Or have the downstream projects been prepared for this change already?

Reply:
Yes, this will cause regression before the changes in downstream project be merged so we need to coordinate the changes.

Reply:
Could the implementation here be made to match the query that was done for `IfMethodHandleInvokes` to prevent the regression? We could then change this back to 'nothing' once the downstream updates are done.

Reply:
Added back the changes to check if inlined callee has methodhandle invoke as suggested.